modules/core/arrow-typeclasses/src/main/kotlin/arrow/typeclasses/MonadContinuations.kt

```diff
@@ -12,8 +13,8 @@ interface BindingInContextContinuation<in T> : Continuation<T> {
 }
 
 @RestrictsSuspension
-open class MonadContinuation<F, A>(M: Monad<F>, override val context: CoroutineContext = EmptyCoroutineContext) :
-  Continuation<Kind<F, A>>, Monad<F> by M {
+open class MonadContinuation<F, A>(val M: Monad<F>, override val context: CoroutineContext = EmptyCoroutineContext) :
```

Comment:
Why is `M` necessary as a field if it's already `this` via delegation?

Reply:
it shouldn't be necessary. Addressed in https://github.com/arrow-kt/arrow/pull/1263/commits/b971abecd65ad59dd767cfe1f59b044faad0ed2c